identity/webapp/.env.dist

```diff
@@ -14,3 +14,8 @@ AUTH_METHOD=local
 
 # API_BASE_URL=
 # API_KEY=
+
+# Is set to a path prefix in deployed environments
+# e.g. /account
```

Comment:
Would the example be `account` as we add the slash [in prefix.ts](https://github.com/wellcomecollection/wellcomecollection.org/pull/6035/files#diff-24e21ba400e7f865047bb041718b764a5c5bcbf0173aa7383c0367d6dd450768R1). Could be worth not prepending the slash to add fewer layers.